libs/types/src/election.ts

```diff
@@ -324,7 +324,7 @@ export interface BallotStyle {
 export type BallotStyleGroupId = NewType<string, 'BallotStyleGroupId'>;
 export const BallotStyleGroupIdSchema =
   IdSchema as unknown as z.ZodSchema<BallotStyleGroupId>;
-export interface ParentBallotStyle {
+export interface BallotStyleGroup {
```

Comment:
Remove `languages` field from `BallotStyleGroup`? I'm not sure what it would mean so safer to remove.